src/libcore/macros.rs

```diff
@@ -229,14 +229,26 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer.
+/// Calls `write_fmt` function on a writer
```

Comment:
Either works, I guess. I’m ambivalent, but @steveklabnik might have a stronger preference towards one of them.